containers/podman-desktop
# Pull image

### What does this PR do?

The Pull Image page wasn't using the standard NavPage, resulting in a UI that didn't match anywhere else. Changed it to use the standard pattern and updated the form a bit.

Both the Pull Image and Build pages did not have a title if there were no providers, fixed that to match other pages.

The Pull image wasn't showing the terminal output div until you start to pull. Although I don't love it I added the missing tick() so that it matches the Build page and always shows.

### Screenshot/screencast of this PR

Before:
<img width="716" alt="Screenshot 2023-05-23 at 10 58 32 AM" src="https://github.com/containers/podman-desktop/assets/19958075/ab0d1efc-805b-42d9-84f6-29638625410c">

After:
<img width="661" alt="Screenshot 2023-06-02 at 3 29 10 PM" src="https://github.com/containers/podman-desktop/assets/19958075/37f0d74f-de6d-41c2-b124-616d17268511">

### What issues does this PR fix or reference?

N/A

### How to test this PR?

Open Pull Image from the images page. Make sure it still works, and is more consistent with the Build Image page.
